exercises/compute-resources.md

```diff
@@ -0,0 +1,29 @@
+# Exercises: Customizing your Compute Resources
+
+We want to do two things in this practice problem:
+
+1. Switch from SLURM to local (or the other way around)
+
+<details>
+    <summary>Solution</summary>
+    In your YAML file, you can change the `pymor.dask_cluster` to either `slurm` or `local`:
+
+    ```yaml
+    pymor:
+        dask_cluster: "slurm"  # or "local"
+    ```
+</details>
```

Comment:
Just a side note: still needs to get correct formatting when I unfold the details part. Not a blocker, just would be nice to have...

Reply:
Fixed, thanks @mandresm